src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -51,21 +51,36 @@
       + 'this is a form which you must complete to tell us more about the claim.',
       { defendantName: claim.claimData.defendant.name }) }}
   </p>
+  <p>{{ t('{{ defendantName }} has rejected your claim. You’ll have to go to a hearing to resolve it.',
+        { defendantName: claim.claimData.defendant.name }) }}</p>
+
+  <p>{{ t('You need to complete a <a href="{{ url }}">directions questionnaire form</a> to tell us more about the claim.',
+        { url: DashboardPaths.directionsQuestionnairePage.evaluateUri({ externalId: claim.externalId }) }) | safe }}</p>
+
+  <p>{{ t('Your claim will be cancelled if you don’t complete and return the form before 4pm on {{ deadline }}.',
+        { deadline: claim.respondedAt | addDays(19) | date }) }}</p>
```

Comment:
As mentioned earlier, please move deadline calculation to route file.

Reply:
done